src/Deprecated/Conversion/Microsoft.Build.Conversion.csproj

```diff
@@ -7,6 +7,7 @@
     <IsPackable>true</IsPackable>
     <PackageDescription>This package contains the $(MSBuildProjectName) assembly which contains logic for converting projects.  NOTE: This assembly is deprecated.</PackageDescription>
     <IncludeSatelliteOutputInPack>false</IncludeSatelliteOutputInPack>
+    <NoWarn>$(NoWarn);1570;1572;1573;1587</NoWarn>
```

Comment:
If all the docs are properly formatted, now, you should be able to delete this line, right?

Reply:
As mentioned in https://github.com/dotnet/msbuild/pull/5575/commits/fe93fcadd754ad0789aa6614ba4ae2ac19f27d6b, the deprecated projects had another couple hundreds errors to fix so I decided to make a PR first for the non-deprecated code.

Reply:
That sounds reasonable to me, but I'd hesitate to close the issue until those are resolved, too.

Reply:
you're right; removed "Fixes" from the last commit so GH doesn't close the issue.